src/Agent.Worker/Release/Artifacts/JenkinsArtifact.cs

```diff
@@ -441,7 +441,7 @@ private bool IsValidBuild(IGenericHttpClient client, JenkinsArtifactDetails jenk
             var buildUrl =
                     string.Format(
                         CultureInfo.InvariantCulture,
-                        "{0}/job/{1}/{2}",
+                        "{0}/job/{1}/{2}/",
```

Comment:
why the extra `/` matter? :D

Reply:
dot net core recently did a security fix where they started clearing Authorization headers on redirect. https://github.com/dotnet/corefx/pull/32776

Reply:
ok, but why adding the `/` fix this? stop the redirect?

Reply:
Yes, if one hits a jenkins url without trailing slash it will return 302 and redirect to url with the slash.

Reply:
cool. 😭